core-service/src/main/java/com/crmfoodestablishment/coreservice/entity/DishInOrder.java

```diff
@@ -0,0 +1,35 @@
+package com.crmfoodestablishment.coreservice.entity;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Entity
+@Table(name = "dish_order")
+public class DishInOrder {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id", nullable = false, unique = true)
+    private Integer id;
+
+    @OneToMany
+    @JoinColumn(name = "dish_id", nullable = false, referencedColumnName = "id")
+    private List<Dish> dishes;
```

Comment:
Mb one to one link and use Dish rather List ? How list of Dishes combine with field "private Short count;"